yt/visualization/tests/test_callbacks_geographic.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+import pytest
+
+from yt import SlicePlot, load_uniform_grid
+from yt.testing import assert_fname, fake_amr_ds, requires_module
+
+
+@requires_module("cartopy")
+@pytest.mark.parametrize("geometry", ["geographic", "internal_geographic"])
+def test_quiver_callback_geographic(geometry, tmp_path):
+    flds = ("density", "velocity_ew", "velocity_ns")
+    units = ("g/cm**3", "m/s", "m/s")
+
+    ds = fake_amr_ds(fields=flds, units=units, geometry=geometry)
+    tmpfi = str(tmp_path / "geo_quiver.png")
+
+    for ax in ds.coordinates.axis_order:
+        slc = SlicePlot(ds, ax, "density", buff_size=(50, 50))
+        if ax == ds.coordinates.radial_axis:
+            slc.set_width((359.99, 179.99))
+        slc.annotate_quiver(("stream", "velocity_ew"), ("stream", "velocity_ns"))
+        slc.save(tmpfi)
+        assert_fname(tmpfi)
```

Comment:
IO is probably making this test longer than it needs to be. Could we just use `slc.render()` instead ? I think it should be sufficient to validate that no exception is raised when the callback actually gets run.

Reply:
yup, just `render` should be fine, will switch.